e2e/cypress/integration/messaging/MM-T134-visual-verification-of-tooltips_spec.js

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+const timestamp = Date.now();
+
+function verifyChannel(res, verifyExistence = true) {
+    const channel = res.body;
+
+    // # Hover on the channel name
+    cy.get(`#sidebarItem_${channel.name}`).should('be.visible').trigger('mouseover');
+
+    // * Verify that the tooltip is displayed
+    if (verifyExistence) {
+        cy.get('div.tooltip-inner').
+            should('be.visible').
+            and('contain', channel.display_name);
+    } else {
+        cy.get('div.tooltip-inner').should('not.exist');
+    }
+}
```

Comment:
The test is failing at this function for me. Can you please recheck:
![Screenshot 2020-09-29 at 6 20 54 PM](https://user-images.githubusercontent.com/1429138/94560676-9becf800-0280-11eb-93ba-6d34f2b3a888.png)

Reply:
The associated function and code has been removed.